leftwm-core/src/handlers/window_handler.rs

```diff
@@ -318,7 +318,7 @@ fn sane_dimension(config_value: Option<Size>, default_ratio: f32, max_pixel: i32
 }
 
 // Get size and position of scratchpad from config and workspace size.
-fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
+pub fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
```

Comment:
If we are using this outside of this file, maybe we should place it in scratchpad_handler

Reply:
I would argue against moving it to scratchpad_handler as it is in my opinion a place to handle the commands specific for the scratchpad and general properties of the windowing of the scratchpad should be placed somewhere else.

If you still want this, I will move it :smile:

Reply:
Hmm, true. Maybe we should move scratchpad.rs to models. And add this (and future function like that) to there